runtime/commonMain/src/kotlinx/serialization/Polymorphic.kt

```diff
@@ -29,27 +29,31 @@ public object PolymorphicClassDescriptor : SerialClassDescImpl("kotlin.Any") {
 /**
  * This class provides support for multiplatform polymorphic serialization for interfaces and abstract classes.
  *
- * Due to security and reflection usage concerns, all serializable implementations of some abstract class must be registered in advance.
- * However, it allows registering subclasses in runtime, not compile-time. For example, it allows adding additional subclasses to the registry
- * that were defined in a separate module, dependent on the base module with the base class.
+ * To avoid the most common security pitfalls and reflective lookup (and potential load) of an arbitrary class,
+ * all serializable implementations of any polymorphic type must be [registered][SerializersModuleBuilder.polymorphic]
+ * in advance in the scope of base polymorphic type, efficiently preventing unbounded polymorphic serialization
+ * and (de)serialization of an arbitrary type.
```

Comment:
You already have `serialization` word before 'and', why '(de)' is in brackets?

Reply:
It was much better in my head :)